internal/dag/builder.go

```diff
@@ -229,7 +220,7 @@ func (b *builder) lookupSecret(m meta, validate func(*v1.Secret) bool) *Secret {
 }
 
 func (b *builder) lookupVirtualHost(name string) *VirtualHost {
-	l := b.listener(b.externalInsecurePort())
+	l := b.listener(80)
```

Comment:
Won't this break if we wanted to serve on a different port, say 8080 since the listener is now hardcoded to 80/443?

Reply:
I thought that too, but this doesn't actually configure the listener at all! It's just a magic number that chooses whether or not the listener in the dag is the secure or insecure one. The actual listening ports are determined purely by the Ingress object. That's why I opened #1135, to clean this up.

Reply:
I've tested this with Ingress documents that listen on random ports, and it all works just fine.

Reply:
:+1:  :tada: